content/en/blog/_posts/2021-04-08-PodSecurityPolicy-Past-Present-and-Future.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: blog
+title: "PodSecurityPolicy: Past, Present, and Future"
+date: 2021-04-08
+slug: PodSecurityPolicy-Past-Present-and-Future
+---
+
+**Author:** Tabitha Sable (Kubernetes SIG Security)
+
+PodSecurityPolicy (PSP) is being deprecated in Kubernetes 1.21. This starts the countdown to its removal, but doesn’t change anything else. PodSecurityPolicy will continue to be fully functional for several more releases before being removed completely. In the meantime, we are developing a replacement for PSP that covers key use cases more easily and sustainably.
```

Comment:
Should we mention “released today” in connection with the Kubernetes 1.21 reference? The current article date is for the release day.